app/models/game.rb

```diff
@@ -17,13 +17,18 @@ def populate!
   end
 
   def check?(color)
-    king = king(color)
+    king = pieces.find_by(type: 'King', color: color)
     enemy_pcs(color).each do |p|
       return true if p.valid_move?(king.x_position, king.y_position)
     end
     false
   end
 
+  # selects enemy pieces that are not captured
+  def enemy_pcs(color)
```

Comment:
I recommend not using erroneous abbreviations in code - it makes it so that you have to mentally try to figure out what abbreviations (if any) you used for things. Not a blocker or anything, just a best practice.